.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: php
+
+sudo: false
+
+git:
+  depth: false
+
+php:
+  - '7.3'
+  - '7.4snapshot'
+  - 'nightly'
+
+env:
+  - DEPENDENCIES=""
+  - DEPENDENCIES="--prefer-lowest --prefer-stable"
+
+matrix:
+  allow_failures:
+  - php: '7.4snapshot'
+  - php: 'nightly'
+
+before_script:
+  - git checkout -b built-ci-branch
+  - composer update --prefer-dist $DEPENDENCIES
+
+script:
+  - vendor/bin/phpunit --coverage-text
```

Comment:
To save time, you can add ` --coverage-xml=coverage/coverage-xml --log-junit=coverage/phpunit.junit.xml`here and add ` --coverage=coverage` to the infection part, so it uses the generated coverage.

Infection needs xml and junit coverage to work

Reply:
Also xdebug can then be disabled after this part, saving some runtime.

Reply:
Nice 👍